.config/test-e2e.js

```diff
@@ -34,6 +36,17 @@ module.exports.create = function create(envArgs) {
       },
     ];
 
+    // TODO remove this
+    c.devServer = {
+      hot: true,
+      writeToDisk: true
+    }
+
+    // TODO remove this
```

Comment:
Are these entries needed?

Reply:
Makes my tdd easier, I'll remove the changes to this file before the final merge.

Reply:
I think that we are close to the final merge. Can we remove that TODOs? The changes seem to be reasonable but for a separate issue.

Reply:
https://github.com/handsontable/handsontable/pull/7648/commits/d52f594df2eca15f8417de08fe2452794ac97554